guides/source/active_record_callbacks.md

```diff
@@ -57,7 +71,31 @@ class User < ApplicationRecord
 end
 ```
 
-Callbacks can also be registered to only fire on certain life cycle events:
+Alternatively you can pass a proc to the callback to be triggered.
+
+```ruby
+class User < ApplicationRecord
+  before_create Proc.new { |user| user.name = user.login.capitalize if user.name.blank? }
+end
+```
+
+Lastly, you can define your own custom callback object, which we will cover later in more detail [below](#callback-classes).
+
+```ruby
+class User < ApplicationRecord
+  before_create MaybeAddName.new
+end
+
+class MaybeAddName
+  def self.before_create(record)
```

Comment:
If we are passing an instance of `MaybeAddName` into the `before_create`, then this should be an instance method and not a class method.

Reply:
@lsylvester Good catch! Fixed in [1a21fc3](https://github.com/rails/rails/pull/46996/commits/1a21fc3bfcd9217699d2d2a89dafede0399114d6) :bow: